mex/backend/rules/helpers.py

```diff
@@ -23,22 +22,21 @@
 }
 
 
-def _transform_graph_result_to_rule_set_response(
-    graph_result: Result,
+def transform_raw_rules_to_rule_set_response(
+    raw_rules: list[dict[str, Any]],
 ) -> AnyRuleSetResponse:
-    """Transform the results of fetching a rule triple into a rule set response."""
+    """Transform a set of plain rules into a rule set response."""
     stem_types: list[str] = []
     stable_target_ids: list[str] = []
     response: dict[str, Any] = {}
     model: type[AnyRuleModel] | None
-    rules = graph_result.one()["items"]
 
-    if len(rules) != 3:
+    if len(raw_rules) != 3:
         raise MExError("inconsistent rule item count")
 
-    for rule in rules:
+    for rule in raw_rules:
         for field_name, model_class_lookup in MODEL_CLASS_LOOKUP_BY_FIELD_NAME.items():
-            if model := model_class_lookup.get(rule.get("entityType")):
+            if model := model_class_lookup.get(str(rule.get("entityType"))):
```

Comment:
when was it not a string?

Reply:
This is merely to silence mypy, due to the type of rule being `dict[str, Any]`.